lib/discourse.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Discourse do
```

Comment:
~~ this module isn't required

Reply:
Mostly have it in for docs and future functionality.

I can remove it, but it might be better to add some docs in stead.